var/spack/repos/builtin/packages/rtags/package.py

```diff
@@ -10,8 +10,10 @@ class Rtags(CMakePackage):
     """RTags is a client/server application that indexes C/C++ code"""
 
     homepage = "https://github.com/Andersbakken/rtags/"
-    url      = "https://andersbakken.github.io/rtags-releases/rtags-2.17.tar.gz"
+    url      = "https://github.com/Andersbakken/rtags/releases/download/v2.38/rtags-2.38.tar.gz"
+    maintainers = ['vmiheer']
 
+    version('2.38', sha256='e19d9cf5823cccc43266ca57c19ae0bb879cbe138511cb3f0343958860481a5d')
     version('2.20', sha256='dceab009194bcfa4265950dac16832bae7883e95d3bc41b215e90bc888db9cb1')
     version('2.17', sha256='cde8882aceb09d65690007e214cc1979e0105842beb7747d49f79e33ed37d383')
```

Comment:
If you're going to change the URL, you'll need to change the sha256 for the older versions as it seems the tar files may be different:

```suggestion
    version('2.38', sha256='e19d9cf5823cccc43266ca57c19ae0bb879cbe138511cb3f0343958860481a5d')
    version('2.20', sha256='9d73399421327147dc47b0ae5b95e12e8355f30291ad1954a78c0ef68b4b501f')
    version('2.17', sha256='288fa49fedf647fb15e2ef10f0ebcd9de1a4ef1bbae3a3940870e136d32a3a60')
```

Options for providing a version method are discussed at https://spack.readthedocs.io/en/latest/packaging_guide.html#version-urls.